zcash_client_sqlite/CHANGELOG.md

```diff
@@ -7,6 +7,22 @@ and this library adheres to Rust's notion of
 
 ## [Unreleased]
 
+### Added
+- `zcash_client_sqlite::AccountUuid`
+- `zcash_client_sqlite::Account::uuid`
+- `zcash_client_sqlite::WalletDb::get_account_for_uuid`
+
+### Changed
+- The `v_transactions` view has been modified:
+  - The `account_id` column has been replaced with `account_uuid`.
+- The `v_tx_outputs` view has been modified:
+  - The `from_account_id` column has been replaced with `from_account_uuid`.
+  - The `to_account_id` column has been replaced with `to_account_uuid`.
+
+### Removed
+- `zcash_client_sqlite::AccountId::{from_u32, as_u32}` (use `AccountUuid` and
+  its methods instead).
```

Comment:
I'm not sure whether this is going to cause problems in the short term for clients that were using the account ids (e.g. to do a release that migrates to UUIDs for persisted data not directly controlled by `zcash_client_sqlite`). The alternative would be to deprecate these methods.

Reply:
We added these behind an unstable feature flag in literally the last release. I do not believe anyone is actually depending on them yet. And anyone who is likely is also using SQL directly for the views, and can join on the `accounts` table for whatever migration they need.

Reply:
We added these behind an unstable feature flag in literally the last release. I do not believe anyone is actually depending on them yet. And anyone who is likely is also using SQL directly for the views, and can join on the `accounts` table for whatever migration they need.

Reply:
Ah I didn't realize they were unstable. That's fine then.